src/MobileRedirection.php

```diff
@@ -481,10 +524,7 @@ private function sanitize_script_attributes( $attributes ) {
 	 * @link https://developers.google.com/search/mobile-sites/mobile-seo/separate-urls#annotation-in-the-html
 	 */
 	public function add_mobile_alternative_link() {
-		printf(
-			'<link rel="alternate" type="text/html" media="only screen and (max-width: 640px)" href="%s">',
-			esc_url( $this->get_current_amp_url() )
-		);
+		amp_add_alternate_link( $this->get_current_amp_url() );
```

Comment:
See note above about duplicate `link` being output currently. If we do what is suggested there, this change could be reverted and the helper function removed because then there would only be one place that this tag is printed anymore.